### PR TITLE
➕ Add stylelint

### PR DESCRIPTION
**Pull request by JuKahl**
- Installed stylelint with `npm install --save-dev stylelint stylelint-config-standard`
- Created `.stylelintrc.json`

- Installed `stylelint-config-prettier`
- Added script `stylelint`& `stylelint-check`

https://stylelint.io/user-guide/get-started#getting-started
https://github.com/prettier/stylelint-config-prettier

### CODE REVIEW

**Comment by RonjaMeinecke** [comment]
no past tense in pull requets 😉

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "build": "next build",
     "start": "next start",
     "eslint": "eslint .",
-    "prettify": "prettier --write ."
+    "prettify": "prettier --write .",
+    "stylelint": "stylelint \"**/*.css\"",
+    "stylelint-check": "stylelint-config-prettier-check"
```
Comment (christophvogel) [inline package.json:12]:
```suggestion
    "stylelint-check": "stylelint-config-prettier-check"
```
not sure if we need this line.

**Review by christophvogel — APPROVED** [review]
GJ 👍

**Review by m-wiemers — APPROVED** [review]
really gj 🚀

File: package.json (hunk 1 above)
Comment (lmachens) [inline package.json:12]: Not sure if this script is useful :)

**Review by lmachens — APPROVED** [review]
gj

**Pull request merged**